parcellab/monolith/values.yaml

```diff
@@ -105,6 +105,32 @@ cronjobs:
   #   successfulJobsHistoryLimit: 3
   #   suspend: false
 
+##
+## Jobs / Hooks
+##
+
+# Only for the defaults
+job:
+  enabled: true
+  job:
+    backoffLimit: 1
+    activeDeadlineSeconds: 300
```

Comment:
```suggestion
  backoffLimit: 1
  activeDeadlineSeconds: 300
```

Reply:
@darioblanco copy & paste, was like that in cronjob. updated the pr to unnest the variables.

Reply:
fixed